module/metrics/execution.go

```diff
@@ -726,10 +755,24 @@ func (ec *ExecutionCollector) RuntimeTransactionInterpreted(dur time.Duration) {
 	ec.transactionInterpretTime.Observe(float64(dur))
 }
 
-// ChunkDataPackRequested is executed every time a chunk data pack request is arrived at execution node.
-// It increases the request counter by one.
-func (ec *ExecutionCollector) ChunkDataPackRequested() {
-	ec.totalChunkDataPackRequests.Inc()
+// ChunkDataPackRequestProcessed is executed every time a chunk data pack request is picked up for processing at execution node.
+// It increases the request processed counter by one.
+func (ec *ExecutionCollector) ChunkDataPackRequestProcessed() {
+	ec.chunkDataPackRequestProcessedTotal.Inc()
+}
+
+// ChunkDataPackResponseDispatchedInNetwork is executed every time a chunk data pack request is successfully replied.
+// It records the time it took to dispatch the reply on the network.
+// It also increases the response counter by one.
+func (ec *ExecutionCollector) ChunkDataPackResponseDispatchedInNetwork(dispatchTime time.Duration) {
+	ec.chunkDataPackResponseDispatchTime.Observe(float64(dispatchTime.Milliseconds()))
```

Comment:
```suggestion
	ec.chunkDataPackResponseDispatchTime.Observe(float64(dispatchTime.Seconds()))
```
Consistency with prometheus naming

Reply:
@jordanschalm I think you mean to refactor it either to `ec.chunkDataPackResponseDispatchTime.Observe(float64(dispatchTime.Seconds()))` or `ec.chunkDataPackResponseDispatchTime.Observe(float64(float64(dispatchTime.Milliseconds())/1000)`, right?

Reply:
Exactly, that's what I meant - thanks for catching!

Reply:
https://github.com/onflow/flow-go/pull/2951/commits/383805d7d94200444432dea05d205138884636ed